Adding `kglab` from PyPI

sugatoray opened the pull request:
<!--
Thank you very much for putting in this recipe PR!

This repository is very active, so if you need help with
a PR or once it's ready for review, please let the right people know.
There are language-specific teams for reviewing recipes.

Currently available teams are:
- python `@conda-forge/help-python`
- python/c hybrid `@conda-forge/help-python-c`
- r `@conda-forge/help-r`
- java `@conda-forge/help-java`
- nodejs `@conda-forge/help-nodejs`
- c/c++ `@conda-forge/help-c-cpp`
- perl `@conda-forge/help-perl`
- Julia `@conda-forge/help-julia`
- ruby `@conda-forge/help-ruby`

If your PR doesn't fall into those categories please contact
the full review team `@conda-forge/staged-recipes`.

Due to GitHub limitations first time contributors to conda-forge are unable
to ping these teams. You can [ping the team][1] using a special command in
a comment on the PR to get the attention of the `staged-recipes` team. You can
also consider asking on our [Gitter channel][2] if your recipe isn't reviewed promptly.

[1]: https://conda-forge.org/docs/maintainer/infrastructure.html#conda-forge-admin-please-ping-team
[2]: https://gitter.im/conda-forge/conda-forge.github.io
-->

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipes/kglab```) and found it was in an excellent condition.

sugatoray commented:
## Build Error Notes

[(error)](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=449784&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=1006)

```sh
The reported errors are:
- Encountered problems while solving:
-   - nothing provides requested chocolate >=0.0.2
-   - nothing provides requested torch >=1.8
-   - nothing provides requested pslpython >=2.2.2
-   - nothing provides requested csvwlib >=0.3.2
```

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipes/chocolate```, ```recipes/kglab```) and found it was in an excellent condition.

sugatoray commented:
## Errors about missing requirements files

### missing: `requirements.txt`

[(error)](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=449796&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=1754)

```sh
Processing $SRC_DIR
  Added file://$SRC_DIR to build tracker '/tmp/pip-req-tracker-rdl4m8pp'
  Running setup.py (path:$SRC_DIR/setup.py) egg_info for package from file://$SRC_DIR
  Running command python setup.py egg_info
  Created temporary directory: /tmp/pip-pip-egg-info-ejsj_gnv
  Preparing metadata (setup.py): started
  Traceback (most recent call last):
    File "<string>", line 1, in <module>
    File "/home/conda/staged-recipes/build_artifacts/kglab_1643469432366/work/setup.py", line 51, in <module>
      base_packages = parse_requirements_file("requirements.txt")
    File "/home/conda/staged-recipes/build_artifacts/kglab_1643469432366/work/setup.py", line 41, in parse_requirements_file
      with pathlib.Path(filename).open() as f:
    File "/home/conda/staged-recipes/build_artifacts/kglab_1643469432366/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/lib/python3.10/pathlib.py", line 1117, in open
      return self._accessor.open(self, mode, buffering, encoding, errors,
  FileNotFoundError: [Errno 2] No such file or directory: 'requirements.txt'
  Preparing metadata (setup.py): finished with status 'error'
WARNING: Discarding file:///home/conda/staged-recipes/build_artifacts/kglab_1643469432366/work. Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

### missing: `requirements-dev.txt`

[(error)](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=449798&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=1773)

```sh
Processing $SRC_DIR
  Added file://$SRC_DIR to build tracker '/tmp/pip-req-tracker-oh4_gg6k'
  Running setup.py (path:$SRC_DIR/setup.py) egg_info for package from file://$SRC_DIR
  Running command python setup.py egg_info
  Created temporary directory: /tmp/pip-pip-egg-info-ps2_8kct
  Preparing metadata (setup.py): started
  Traceback (most recent call last):
    File "<string>", line 1, in <module>
    File "/home/conda/staged-recipes/build_artifacts/kglab_1643469951442/work/setup.py", line 52, in <module>
      docs_packages = parse_requirements_file("requirements-dev.txt")
    File "/home/conda/staged-recipes/build_artifacts/kglab_1643469951442/work/setup.py", line 41, in parse_requirements_file
      with pathlib.Path(filename).open() as f:
    File "/home/conda/staged-recipes/build_artifacts/kglab_1643469951442/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/lib/python3.10/pathlib.py", line 1117, in open
      return self._accessor.open(self, mode, buffering, encoding, errors,
  FileNotFoundError: [Errno 2] No such file or directory: 'requirements-dev.txt'
WARNING: Discarding file:///home/conda/staged-recipes/build_artifacts/kglab_1643469951442/work. Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
  Preparing metadata (setup.py): finished with status 'error'
```
